--- cm_library_c_binding/CMakeLists.txt.orig 2024-06-27 15:32:12.619396727 -0500
+++ cm_library_c_binding/CMakeLists.txt     2024-06-27 15:33:29.757155052 -0500
@@ -35,3 +35,4 @@
 )

 add_custom_target(cffi-c-binding ALL DEPENDS ${CFFI_C_CODE_DIR}/${CFFI_C_CODE})
+add_dependencies(cffi-c-binding headers-for-cffi)
--- cm_python_module/CMakeLists.txt.orig 2024-06-27 15:32:12.619396727 -0500
+++ cm_python_module/CMakeLists.txt     2024-06-27 15:33:29.757155052 -0500
@@ -23,9 +23,10 @@
     # https://docs.python.org/3/c-api/stable.html#limited-c-api
     target_link_libraries(${CFFI_OUTPUT_LIBRARY} PRIVATE ${CFFI_INPUT_LIBRARY})
 elseif(VENDORED_AS_SYSTEM_LIB_FOUND)
-    message(STATUS "Vendored system library found: ${VENDORED_AS_SYSTEM_LIB_LINK_LIBRARIES}")
+    message(STATUS "Vendored system library found: ${VENDORED_AS_SYSTEM_LIB_LIBRARIES}")
+    add_dependencies(${CFFI_OUTPUT_LIBRARY} cffi-c-binding)
     target_include_directories(${CFFI_OUTPUT_LIBRARY} PRIVATE ${VENDORED_AS_SYSTEM_LIB_INCLUDE_DIRS})
-    target_link_libraries(${CFFI_OUTPUT_LIBRARY} PRIVATE ${VENDORED_AS_SYSTEM_LIB_LDFLAGS})
+    target_link_libraries(${CFFI_OUTPUT_LIBRARY} PRIVATE ${VENDORED_AS_SYSTEM_LIB_LIBRARIES})
 else()
     message(FATAL_ERROR "Vendored library not found.")
 endif()
--- cmake/SetSystemLibIfExists.cmake.orig 2024-06-27 15:32:12.619396727 -0500
+++ cmake/SetSystemLibIfExists.cmake     2024-06-27 15:33:29.757155052 -0500
@@ -1,5 +1,6 @@
 function (SetSystemLibIfExists)
-    set(_paths "$ENV{PKG_CONFIG_PATH}" "$ENV{CONDA_PREFIX}/Library/lib/pkgconfig" "$ENV{CONDA_PREFIX}/lib/pkgconfig")
-    cmake_path(CONVERT "${_paths}" TO_NATIVE_PATH_LIST _paths)
-    set(ENV{PKG_CONFIG_PATH} ${_paths})
+    set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH};$ENV{CONDA_PREFIX}/Library/lib/pkgconfig;$ENV{CONDA_PREFIX}/lib/pkgconfig")
+    if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
+        set(PKG_CONFIG_EXECUTABLE "${PKG_CONFIG_EXECUTABLE};--msvc-syntax;--dont-define-prefix")
+    endif()

